### PR TITLE
Increase the number of parameters obtained during learning #364

### LINKED ISSUE #364: Increase the number of parameters obtained during learning
When the labeled segments get in the analytics, we need to extract more information for training.
For example, the absolute value of the height, wigth, correlation coefficient between the patterns.

### PR DESCRIPTION
**Pull request by VargBurz**
closes #364

### CODE REVIEW

File: analytics/analytics/models/drop_model.py [hunk 1]
```diff
@@ -32,27 +32,32 @@ def do_fit(self, dataframe: pd.DataFrame, labeled_segments: list, deleted_segmen
...
         convolve_list = []
+        correlation_list = []
         drop_height_list = []
         drop_length_list = []
         patterns_list = []
-
+        pattern_timestamp = []
         for segment in labeled_segments:
-            confidence = utils.find_confidence(segment.data)
+            confidence = utils.find_confidence(segment.data)[0]
```
Comment (rozetko) [inline analytics/analytics/models/drop_model.py]: What if `utils.find_confidence(segment.data)` returns empty result?

File: analytics/analytics/utils/common.py [hunk 2]
```diff
@@ -154,11 +155,47 @@ def nan_to_zero(segment: Union[pd.Series, list], nan_list: list) -> Union[pd.Ser
...
     return segment
 
-def find_confidence(segment: pd.Series) -> float:
+def find_confidence(segment: pd.Series) -> [float, float]:
     segment = utils.check_nan_values(segment)
     segment_min = min(segment)
     segment_max = max(segment)
-    return CONFIDENCE_FACTOR * (segment_max - segment_min)
+    height = segment_max - segment_min
+    return CONFIDENCE_FACTOR * height, height
```
Comment (rozetko) [inline analytics/analytics/utils/common.py]: Maybe it would be better to return dict? To give name to returned values

Comment (jonyrock) [reply analytics/analytics/utils/common.py]: Lets 
* return `(float, float)` -- which is Python pair,
* add comment to the function with `'''` comment what it does. It is general advise to write comments to functions in utils

File: analytics/analytics/models/jump_model.py [hunk 3]
```diff
@@ -33,26 +33,32 @@ def do_fit(self, dataframe: pd.DataFrame, labeled_segments: list, deleted_segmen
...
         confidences = []
         convolve_list = []
+        correlation_list = []
         jump_height_list = []
         jump_length_list = []
         patterns_list = []
+        pattern_timestamp = []
         for segment in labeled_segments:
-            confidence = utils.find_confidence(segment.data)
+            confidence = utils.find_confidence(segment.data)[0]
```
[the remark this answers is not in the document]
Comment (VargBurz) [reply analytics/analytics/models/jump_model.py]: If there are any problems in the method `utils.find_confidence(segment.data)` it  will return `(0, 0)`

File: analytics/analytics/models/peak_model.py [hunk 4]
```diff
@@ -32,26 +32,40 @@ def do_fit(self, dataframe: pd.DataFrame, labeled_segments: list, deleted_segmen
...
+        delete_pattern_height = []
+        delete_pattern_timestamp = []
         for segment in deleted_segments:
             del_max_index = segment.data.idxmax()
+            delete_pattern_timestamp.append(dataframe['timestamp'][del_max_index])
             deleted = utils.get_interval(data, del_max_index, self.state['WINDOW_SIZE'])
             deleted = utils.subtract_min_without_nan(deleted)
             del_conv = scipy.signal.fftconvolve(deleted, self.model)
             if len(del_conv): del_conv_list.append(max(del_conv))
+            delete_pattern_height.append(utils.find_confidence(deleted)[1])
```
[the remark this answers is not in the document]
Comment (VargBurz) [reply analytics/analytics/models/peak_model.py]: same

File: analytics/analytics/models/peak_model.py [hunk 5]
```diff
@@ -32,26 +32,40 @@ def do_fit(self, dataframe: pd.DataFrame, labeled_segments: list, deleted_segmen
...
-            confidence = utils.find_confidence(segment.data)
+            confidence = utils.find_confidence(segment.data)[0]
             confidences.append(confidence)
             segment_max_index = segment.data.idxmax()
             self.ipeaks.append(segment_max_index)
+            pattern_timestamp.append(dataframe['timestamp'][segment_max_index])
             labeled = utils.get_interval(data, segment_max_index, self.state['WINDOW_SIZE'])
             labeled = utils.subtract_min_without_nan(labeled)
             patterns_list.append(labeled)
+            pattern_height.append(utils.find_confidence(labeled)[1])
```
[the remark this answers is not in the document]
Comment (VargBurz) [reply analytics/analytics/models/peak_model.py]: same

**Pull request merged**